geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -601,25 +602,36 @@ public Result preExecution(GfshParseResult parseResult) {
         }
       }
 
-      String statisticsEnabled =
-          parseResult.getParamValueAsString(CliStrings.CREATE_REGION__STATISTICSENABLED);
-      if (!Boolean.parseBoolean(statisticsEnabled)) {
-        String entryIdle =
-            parseResult.getParamValueAsString(CliStrings.CREATE_REGION__ENTRYEXPIRATIONIDLETIME);
-        String entryTtl =
-            parseResult.getParamValueAsString(CliStrings.CREATE_REGION__ENTRYEXPIRATIONTIMETOLIVE);
-        String regionIdle =
-            parseResult.getParamValueAsString(CliStrings.CREATE_REGION__REGIONEXPIRATIONIDLETIME);
-        String regionTtl =
-            parseResult.getParamValueAsString(CliStrings.CREATE_REGION__REGIONEXPIRATIONTTL);
-        if (entryIdle != null || entryTtl != null || regionIdle != null || regionTtl != null) {
-          String message =
-              LocalizedStrings.AttributesFactory_STATISTICS_MUST_BE_ENABLED_FOR_EXPIRATION
-                  .toLocalizedString();
-          return ResultBuilder.createUserErrorResult(message + ".");
-        }
+      // if any expiration value is set, statistics must be enabled
```

Comment:
For readability, I might drop this to another method so it could read as
```
if (expirySetWithoutStatistics(parseResult)) {
  return ResultBuilder.yadaYada();
}
```

Then again, I kind of think that's true of every major block in this `preexecution`.  And I think the `Interceptor` should be its own class instead of a subclass... So this probably falls into "We should refactor Interceptors" and not a problem with this ticket.

Reply:
agree, another refactor effort.